studio/components/interfaces/Billing/Invoices.tsx

```diff
@@ -117,8 +122,23 @@ const Invoices: FC<Props> = ({ projectRef }) => {
                       <Table.td>
                         <p>{x.number}</p>
                       </Table.td>
+                      <Table.td>
+                        <InvoiceStatusBadge status={x.status} />
+                      </Table.td>
                       <Table.td className="align-right">
                         <div className="flex items-center justify-end space-x-2">
+                          {['uncollectible', 'open'].includes(x.status) && (
+                            <Button
+                              as="a"
```

Comment:
like wise, wrap this within a

import Link from 'next/link'

```
<Link href="xxx">
  <a target="_blank">
    <Button />
  <a/>
</Link>
```

then remove the href on the button